collectfast/tests/utils.py

```diff
@@ -64,10 +64,10 @@ def test(func: F) -> Type[unittest.TestCase]:
     return test
 
 
-def create_static_file() -> pathlib.Path:
+def create_static_file(size: int = 500) -> pathlib.Path:
     """Write random characters to a file in the static directory."""
     path = static_dir / f"{uuid.uuid4().hex}.txt"
-    path.write_text("".join(chr(random.randint(0, 64)) for _ in range(500)))
+    path.write_text("".join(chr(random.randint(0, 64)) for _ in range(size)))
```

Comment:
I noticed this takes some time to generate the file now. It can be made fast with something like this:

```python
def create_static_file(size: int = 500) -> pathlib.Path:
    """Write random characters to a file in the static directory."""
    path = static_dir / f"{uuid.uuid4().hex}.txt"
    with path.open("wb") as handle:
        handle.write(os.urandom(size))
    return path
```

Reply:
Cool, I was wondering if it would be really slow to generate a 10MB file using the current code. I'll make this change.

Reply:
Took about 7s with the current code on my machine, and about 200ms with this improvement. So not a big problem anyways, but it's nice to have quick tests :)

Reply:
Done in 5f3fea8.